app/models/enterprise.rb

```diff
@@ -357,6 +383,12 @@ def inventory_variants
     end
   end
 
+  def distributed_properties
+    (distributed_product_properties + distributed_producer_properties).uniq do |property_object|
+      property_object.property.presentation
+    end
+  end
```

Comment:
As I mentioned in the PR description, this could potentially be done one the DB level too because it might make things simpler - but I suppose I don't have enough knowledge on how this is used so far